README.md

```diff
@@ -1,6 +1,6 @@
-# debugger.html
+# Debugger.html
```

Comment:
haha - yeah, wanted to have a conversation about and hear why. sorry forgot to comment